pkg/volume/flexvolume/util.go

```diff
@@ -140,7 +140,7 @@ func prepareForMount(mounter mount.Interface, deviceMountPath string) (bool, err
 // Mounts the device at the given path.
 // It is expected that prepareForMount has been called before.
 func doMount(mounter mount.Interface, devicePath, deviceMountPath, fsType string, options []string) error {
-	err := mounter.Mount(devicePath, deviceMountPath, fsType, options)
+	err := mounter.MountSensitiveWithoutSystemd(devicePath, deviceMountPath, fsType, options, nil)
```

Comment:
It's possible flex volumes can use fuse right?

Reply:
@msau42 you mean Linux fuse file system? why is this PR related?

Reply:
systemd is needed to mount storage backends that spawn fuse daemons. If we think flex volumes could use fuse, then we need to keep it using systemd.

Reply:
there is still flexvolume driver that use fuse: https://github.com/Azure/kubernetes-volume-drivers/tree/master/flexvolume/blobfuse,  this will break blobfuse flexvol driver?

Reply:
`doMount` in flex volume plugin is used only to bind-mount volume into pod-local directory in case a flex driver does not implement its own `mount` call. I assume such driver implements `mountdevice` and it can use systemd as it wants there.

See `-o bind` forcefully added here:
https://github.com/kubernetes/kubernetes/blob/442a69c3bdf6fe8e525b05887e57d89db1e2f3a5/pkg/volume/flexvolume/mounter-defaults.go#L37

I think it is safe to bind mount without systemd.

Reply:
THX @jsafrane yes bind mount not use systemd